src/main/java/com/answerdigital/answerking/model/Tag.java

```diff
@@ -14,13 +14,15 @@
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
+import lombok.Builder;
 
 import java.util.List;
 
 @Entity
 @Table(name = "tag")
 @Getter
 @Setter
+@Builder
```

Comment:
Can we please remove this annotation? It isn't being used. I've got a note on my TODO list to make a ticket to remove these from the entire codebase.